### DISCUSSION TITLE
How to include your collection's documentation to https://docs.ansible.com/ansible/latest/collections/index.html

### DISCUSSION THREAD

**Discussion by bhati-pradeep** [ansible-collections/overview#227] [Q&A]
Hi team, 
Can someone lead me to the process for hosting any collection's documentation on ansible doc site similar to other collections here https://docs.ansible.com/ansible/latest/collections/index.html.

Thanks

**Comment by mariolenz** [comment]
I'm not 100% sure, but I think the documentation there is only about collections that are part of the [community package](https://pypi.org/project/ansible/). That is if your collection isn't part of it, it won't show up there. If it is, it should do so automatically. I dimly remember discussions about this, but don't find them at the moment.

BTW: At the moment, there's a lot of work done to create a new community website. I'm not sure if it will allow collections to publish their documentation there if they are _not_ part of the community package. But maybe it's planned.

cc @cybette @GregSutcliffe

**Reply by felixfontein** [reply]
The collections whose documentation shows up on https://docs.ansible.com/ansible/latest/collections/index.html are exactly the collections that are included in the Ansible community package. That is because https://docs.ansible.com/ansible/latest/ **is** the documentation page for the latest version of Ansible community package :-)

**Comment by bhati-pradeep** [comment]
Collections that are under ansible-collections ? I saw few which were not and still have docs there.

**Reply by felixfontein** [reply]
No, collections included in the Ansible community package. See https://github.com/ansible-community/ansible-build-data/blob/main/8/ansible.in for the collections included in the Ansible community package (version 8; currently `/latest/` refers to 7 but soon it will refer to 8).

If you are interested in publishing collection documentation similar to the docs on https://docs.ansible.com/ansible/latest/collections/, take a look at https://github.com/ansible-community/antsibull-docs/. It is used to generate the collection documentation on https://docs.ansible.com/ansible/latest/collections/.

**Reply by bhati-pradeep** [reply]
For publishing our collection docs in https://docs.ansible.com/ansible/latest/collections/ , our collection needs to be in Ansible Community package, right ? Whats the process to get collection part of Ansible Community package ?

**Reply by felixfontein** [reply]
See https://github.com/ansible-collections/ansible-inclusion/. (Note that you **should not** apply for inclusion just to get your collection docs listed on https://docs.ansible.com/ansible/latest/collections/.)

**Reply by mariolenz** [reply]
I agree with @felixfontein, you **should not** apply for inclusion just to get your collection docs listed on https://docs.ansible.com/ansible/latest/collections/

**Comment by Andersson007** [comment]
Closing as the discussion hasn't been active for a long time. Thanks everyone!
